Sources/iron/data/SceneFormat.hx

```diff
@@ -561,3 +562,19 @@ typedef TTrack = {
 	public var values: Float32Array; // sampled - full matrix transforms, non-sampled - values
 	@:optional public var ref_values: Array<Array<String>>; // ref values
 }
+
+#if js
+typedef TPostprocessData = {
+#else
+@:structInit class TPostprocessData {
+#end
+	@:optional public var bloom: TBloomData;
+}
+
+#if js
+typedef TBloomData = {
+#else
+@:structInit class TBloomData {
+#end
+	public var radius: Float;
```

Comment:
Can we move this to the [config file](https://github.com/armory3d/armory/blob/main/Sources/armory/data/Config.hx#L48)? It probably would not cause any issues if we have `arm_config` always enabled. 🤔Could be also hard-coded at [Main.hx](https://github.com/armory3d/armory/blob/main/blender/arm/write_data.py#L461) like we do for some other stuff if needed?

Amazing stuff!

Reply:
Sure, I can revert that commit. I'm fine with both ways, using the config as well as Main.hx. Do you have a preference? Using the config makes more sense I think, but it is a little less lightweight and if I interpret the code correctly, Armory then seems to compile some shader variants which might slow down compilation by a bit. It's probably not a big deal though.

The code related to texture filtering has to remain in Iron though, since setting texture parameters of render targets is not possible from within Armory at the moment if I'm not mistaken.

Reply:
Hm, just adding it to the config [here](https://github.com/armory3d/armory/blob/main/blender/arm/write_data.py#L420) should not introduce shader variants I think. But if it's in the config it should be configurable at runtime, otherwise hardcoding at [Main.hx](https://github.com/armory3d/armory/blob/main/blender/arm/write_data.py#L461) is probably good for now.

Reply:
It's updated now, I decided to go for Main.hx for now since I don't know whether you want to merge this before the 23.01 SDK release and I would want more time to check how the arm_config option is implemented. I can still implement this as part of a future PR of course :)

The reason I'm suspecting arm_config is creating shader variants is [this](https://github.com/armory3d/armory/blob/3a8ae93591aa75f2e1018bf622766bd9d0e8c93f/blender/arm/make.py#L238).